Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -0,0 +1,344 @@
+        else:
+            crowdstrike_types = [f"type:'{XSOAR_TYPES_TO_CROWDSTRIKE.get(type.lower())}'" for type in types_list if
+                                 type.lower() in XSOAR_TYPES_TO_CROWDSTRIKE]
+
+        result = ','.join(crowdstrike_types)
+        return result
+
+
+def fetch_indicators_command(client: Client):
+    """ fetch indicators from the Crowdstrike Intel
+
+    Args:
+        client: Client object
+
+    Returns:
+        list of indicators(list)
+    """
+    parsed_indicators = client.fetch_indicators(
+        fetch_command=True,
+        limit=client.limit
+    )
+    # we submit the indicators in batches
+    for b in batch(parsed_indicators, batch_size=2000):
+        demisto.createIndicators(b)
+    return parsed_indicators
+
+
+def crowdstrike_indicators_list_command(client: Client, args: dict) -> CommandResults:
+    """ Gets indicator from Crowdstrike Intel to readable output
+
+    Args:
+        client: Client object
+        args: demisto.args()
+
+    Returns:
+        readable_output, raw_response
+    """
+
+    offset = arg_to_number(args.get('offset', 0))
+    limit = arg_to_number(args.get('limit', 50))
+    parsed_indicators = client.fetch_indicators(
+        limit=limit,
+        offset=offset,
+        fetch_command=False
+    )
+    if outputs := copy.deepcopy(parsed_indicators):
+        for indicator in outputs:
+            indicator['id'] = indicator.get('rawJSON', {}).get('id')
+
+        readable_output = tableToMarkdown(name='Indicators from CrowdStrike Falcon Intel', t=outputs,
+                                          headers=["type", "value", "id"], headerTransform=pascalToSpace)
+
+        return CommandResults(
+            outputs=outputs,
+            outputs_prefix='CrowdStrikeFalconIntel.Indicators',
+            outputs_key_field='id',
+            readable_output=readable_output,
+            raw_response=parsed_indicators
+        )
+    else:
+        return CommandResults(
+            readable_output='No Indicators.'
+        )
+
+
+def test_module(client: Client, args: dict) -> str:
+    try:
+        client.fetch_indicators(limit=client.limit, fetch_command=False)
+    except Exception:
+        raise Exception("Could not fetch CrowdStrike Indicator Feed\n"
+                        "\nCheck your API key and your connection to CrowdStrike.")
+    return 'ok'
+
+
+def reset_last_run():
+    """
+    Reset the last run from the integration context
+    """
+    demisto.setIntegrationContext({})
+    return CommandResults(readable_output='Fetch history deleted successfully')
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    params = demisto.params()
+
+    credentials = params.get('credentials')
+    proxy = params.get('proxy', False)
+    insecure = params.get('insecure', False)
+    first_fetch = datetime.timestamp(dateparser.parse(params.get('first_fetch'))) if params.get('first_fetch') else None
```

Comment:
```suggestion
    first_fetch = arg_to_datetime(`first_fetch`)
```

Reply:
changed to :   first_fetch = datetime.timestamp(arg_to_datetime(params.get('first_fetch'))) if params.get('first_fetch') else None
since I need timestamp, and tot datetime.

Reply:
outdated